### PR TITLE
Fix SSL connection errors with Java 1.7.

### PR DESCRIPTION
**Pull request by damonkohler**

### CODE REVIEW

**Comment by damonkohler** [comment]
@SirVer PTAL

File: gradle.properties [hunk 1]
```diff
@@ -1 +1,2 @@
 org.gradle.daemon=true
+org.gradle.jvmargs=-Djsse.enableSNIExtension=false
```
Comment (SirVer) [inline gradle.properties:3]: a comment would help.

**Comment by SirVer** [comment]
done.

**Comment by damonkohler** [comment]
Fixed.

**Pull request merged**